src/containers/shared/components/Transaction/XChainModifyBridge/parser.ts

```diff
@@ -0,0 +1,18 @@
+import formatAmount from '../../../../../rippled/lib/txSummary/formatAmount'
+
+export function parser(tx: any, meta: any) {
+  const affectedNodes = meta.AffectedNodes
+  const bridgeMeta = affectedNodes.filter(
+    (node: any) =>
+      node.ModifiedNode && node.ModifiedNode.LedgerEntryType === 'Bridge',
+  )[0]
+  return {
+    lockingDoor: tx.XChainBridge.LockingChainDoor,
+    lockingIssue: tx.XChainBridge.LockingChainIssue,
```

Comment:
Not required for this PR but the XChainBridge parsing could be in a shared utility function.

Reply:
Will do in a future cleanup PR